kedro/io/data_catalog.py

```diff
@@ -144,6 +144,7 @@ def __init__(  # noqa: PLR0913
         dataset_patterns: Patterns | None = None,
         load_versions: dict[str, str] | None = None,
         save_version: str | None = None,
+        default_pattern: Patterns | None = None,
```

Comment:
Can you add this to the docstring as well?

Reply:
done